spec/TestClient.ts

```diff
@@ -55,10 +53,6 @@ export class TestClient implements IE2EKeyReceiver, ISyncResponder {
         sessionStoreBackend?: Storage,
         options?: Partial<ICreateClientOpts>,
     ) {
-        if (sessionStoreBackend === undefined) {
-            sessionStoreBackend = new MockStorageApi() as unknown as Storage;
```

Comment:
This seems unrelated

Reply:
Wee https://github.com/matrix-org/matrix-js-sdk/pull/4682#discussion_r1941150818